### PR TITLE
Add examples for rename

### PR DESCRIPTION
**Pull request by koperagen**

### CODE REVIEW

File: docs/StardustDocs/topics/rename.md [hunk 1]
```diff
@@ -8,3 +10,57 @@ df.rename { columns }.into { nameExpression }
...
+<dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.rename.html"/>
+<!---END-->
+
+
+<!---FUN renameExpression-->
+<tabs>
+<tab title="Properties">
+
+```kotlin
+
```
Comment (zaleslaw) [inline docs/StardustDocs/topics/rename.md:47]: is it ok to have here empty sections?

File: core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/Modify.kt [hunk 2]
```diff
@@ -1464,4 +1464,63 @@ class Modify : TestBase() {
...
+        }
+        // SampleEnd
+    }
+
+    @Test
+    @TransformDataFrameExpressions
+    fun renameExpression_accessors() {
+        // SampleStart
+        val age by column<Int>()
+        df.rename { age }.into {
```
Comment (zaleslaw) [inline core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/Modify.kt:1509]: Should these renameExpressions displayed below in tabs?

**Review by zaleslaw — COMMENTED** [review]
Could you please answer my questions regarding documentation?

File: docs/StardustDocs/topics/rename.md (hunk 1 above)
Comment (Jolanrensen) [reply docs/StardustDocs/topics/rename.md:47]: indeed, looks like korro skipped these @koperagen

Comment (koperagen) [reply docs/StardustDocs/topics/rename.md]: My bad! Fixed it, thank you

**Pull request merged**